src/Microsoft.AspNet.OData.Shared/DefaultODataAPIHandler.cs

```diff
@@ -0,0 +1,157 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="DefaultODataAPIHandler.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// This is the default ODataAPIHandler for CLR type. This calss has default Get, Create and Update
+    /// and will do these actions. This will be used when the original collection to be Patched is provided.
+    /// </summary>
+    /// <typeparam name="TStructuralType"></typeparam>
+    internal class DefaultODataAPIHandler<TStructuralType> : ODataAPIHandler<TStructuralType> where TStructuralType :class
+    {
+        Type _clrType;
+        ICollection<TStructuralType> originalList;
```

Comment:
Consistency... How come one private field is declared with a leading `_` while the other is not?